pkg/vere/db/lmdb.c

```diff
@@ -29,11 +29,24 @@ intmax_t mdb_get_filesize(mdb_filehandle_t han_u);
 //    supported operations are as follows
 //
 //      - open/close an environment
+//      - backup an environment
+//      - delete an environment
 //      - read/save metadata
 //      - read the first and last event numbers
 //      - read/save ranges of events
 //
 
+/* u3_lmdb_backup(): backup the lmdb environment.
+*/
+c3_o u3_lmdb_backup(MDB_env* env_u, const c3_c* pax_c)
+{
+  if ( 0 != mdb_env_copy(env_u, pax_c) ) {
```

Comment:
I took a look at the [implementation](https://github.com/LMDB/lmdb/blob/3947014aed7ffe39a79991fa7fb5b234da47ad1a/libraries/liblmdb/mdb.c#L10517) of this and it uses `write()`, which will be a lot slower than just using `rename()` from `stdio.h`.